data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/metrics/Metrics.java

```diff
@@ -213,4 +219,12 @@ private static String getHost() {
       .baseUnit(BaseUnits.MILLISECONDS)
       .serviceLevelObjectives(LATENCY_SLOs);
   }
+
+  public static Counter.Builder discardedEventCount(final io.micrometer.core.instrument.Tags tags) {
+    return Counter
+      .builder(DISCARDED_EVENTS_COUNT)
+      .description("Number of events discarded")
```

Comment:
nit: since we discard events that don't pass filtering too, to distinguish between the two discarded concepts and clarify that these are only the invalid ones.
```suggestion
      .description("Number of invalid events discarded")
```